### PR TITLE
How I imagine this service to work

### PR DESCRIPTION
**Pull request by qvalentin**
could have also started a discussion :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,31 @@ Base of this code base is this [repo](https://github.com/snoyberg/file-server-de
...
+- [ ] service is either mapped with a usefull prefix /userdata/ or /data/
 **(Roadmap feature)**
 - [ ] there is another path /preview/id?token=token
 
+
+### Upload
+- FE sends POST with /upload, files content, path and token (as Header)
+- FH sends POST with path, token and file information to rest
+- rest creates file in DB, response is 200 and ids of files
+- FH saves files in a folder (named by a hash of the file id) with filename == fileid
```
Comment (open-schnick) [inline README.md:26]: talking about the Fh saving files. The FH is sending a post to the REST endpoint. And the endpoint returns a array of saved file instances. The instance would be a slightly changed version with a "realPath" / "realName" key that specifies how the FH needs to save the file. (when downloading the FH gets the same answer when asked for the files with the same ids.)  
( I would rewrite the endpoints in rest accordingly)

Comment (qvalentin) [reply README.md:26]: Ok so you don't want to store the files with obfuscated names and also want to create the folder structure?

Comment (open-schnick) [reply README.md:26]: obfuscated (which means somehow hashed) names without structure.

Comment (qvalentin) [reply README.md:26]: Why not just use the id as filename then?

Comment (open-schnick) [reply README.md:26]: y doesn't matter really. we are not going to implement this tomorrow

**Pull request closed without merging**